src/i18n/locales/pt-br.json

```
@@ -0,0 +1,99 @@
+    "pages.notFound.takeMeBack": "Voltar para página inicial",
+    "pages.blog.noEntries": "Sem artigos nessa categoria.",
+    "pages.docs.apiVersion.select": "Selecione a versão da API",
+    "pages.index.features.javascript.title": "JavaScript",
+    "pages.index.features.javascript.description": "Node.js fornece suporte para a linguagem de programação JavaScript",
+    "pages.index.features.openSource.title": "Open-Source",
+    "pages.index.features.openSource.description": "Node.js é open-source e ativamente mantido por contribuidores globais",
+    "pages.index.features.everywhere.title": "Multiplataforma",
+    "pages.index.features.everywhere.description": "Node.js foi criado para trabalhar em diversas situações e plataformas distintas",
+    "components.tableOfContents.heading": "Tabela de Conteúdo",
+    "docs.api.addedIn": "Adicionado na versão: {version}",
+    "docs.api.removedIn": "Removido na versão: {version}",
+    "docs.api.deprecatedIn": "Depreciado na versão: {version}",
+    "docs.api.nApiVersion": "Versão da N-API: {version}",
+    "docs.api.history": "Histórico",
+    "docs.api.sourceLink": "Código Fonte:",
+    "docs.api.history.version": "Versão",
+    "docs.api.history.changes": "Mudanças",
+    "docs.api.stability": "Estabilidade: {level}{text}",
+    "components.header.links.about": "Sobre",
+    "components.header.links.certification": "Certificação",
+    "components.author.githubLinkLabel": "{username} Github - Abre em uma nova ABA",
+    "navigation.about.section.about": "Sobre o Node.js",
+    "navigation.about.section.getInvolved": "Participe",
+    "navigation.about.section.download": "Download",
+    "navigation.about.items.collabSummit": "Summit de Colaboradores",
+    "navigation.about.items.contribute": "Contribua",
+    "navigation.about.items.previousReleases": "Versões Anteriores",
+    "navigation.about.items.codeOfConduct": "Código de Conduta",
+    "navigation.about.items.governance": "Governança do Projeto",
+    "navigation.about.items.releases": "Versões",
+    "navigation.about.items.resources": "Recursos",
+    "navigation.about.items.privacy": "Política de Privacidade",
+    "navigation.about.items.security": "Segurança",
+    "navigation.about.items.packageManager": "Gerenciador de Pacotes",
+    "navigation.blog.categories": "Categorias",
+    "blog.categories.all": "Tudo",
+    "blog.categories.announcements": "Anúncios",
+    "blog.categories.release": "Versões",
+    "blog.categories.vulnerability": "Vulnerabilidades",
+    "blog.categories.announcements.description": "Anúncios gerais do Node.js",
+    "blog.categories.release.description": "Anúncios sobre novas versões do Node.Js",
+    "blog.categories.vulnerability.description": "Anúncios envolvendo Vulnerabilidades de Segurança",
+    "components.downloadToggle.ltsVersions": "Versões LTS",
+    "components.downloadToggle.currentVersions": "Versões atuais"
```

let's keep the style pattern in the menu
```suggestion
    "components.downloadToggle.currentVersions": "Versões Atuais"
```